doc/nrf/ug_thingy91.rst

```diff
@@ -0,0 +1,251 @@
+
+The table below shows the different types of build files that are generated and the different scenarios in which they are used:
+
++--------------+----------------------------------------+---------------------------------------+
+| File         | File format                            | Programming scenario                  |
++==============+========================================+=======================================+
+|merged.hex    | Full image, HEX format                 | Using debug probe                     |
++--------------+----------------------------------------+---------------------------------------+
+|app_signed.hex| MCUboot compatible image, HEX format   | Using nRF Connect Programmer          |
++--------------+----------------------------------------+---------------------------------------+
+|app_update.bin| MCUboot compatible image, binary format|* Using ``mcumgr`` command line tool   |
+|              |                                        |* For FOTA updates                     |
++--------------+----------------------------------------+---------------------------------------+
+
+There are multiple methods of programming a sample or application onto a Thingy:91.
+You can choose the method based on the availability or absence of an external debug probe to program.
+
+.. note::
+
+   In the event of not having an external debug probe to program the Thingy:91, you can directly program by :ref:`using the USB (MCUboot) method and nRF Connect Programmer<programming_usb>`.
+   The ``app_signed.hex`` firmware image file is used for the update in this scenario.
+
+.. _build_pgm_segger:
+
+Building and programming using SEGGER Embedded Studio
+=====================================================
+
+.. include:: gs_programming.rst
+   :start-after: build_SES_projimport_open_start
+   :end-before: build_SES_projimport_open_end
+
+
+.. figure:: images/ses_thingy_configuration.png
+   :alt: Opening the AT Client sample
+
+   Opening the AT Client sample for the nrf9160_pca20035ns build target
+
+.. note::
+
+   The *Board Directory* folder can be found in the following location: ``ncs/nrf/boards/arm``.
+
+
+4. Click **OK** to import the project into SES. You can now work with the project in the IDE.
+
+   .. include:: gs_programming.rst
+      :start-after: build_SES_projimport_note_start
+      :end-before: build_SES_projimport_note_end
+
+#. To build the sample or an application:
+
+   a. Select your project in the Project Explorer.
+   #. From the menu, select **Build -> Build Solution**.
+      This builds the project.
+
+   The merged hex file, containing both the application and the SPM is located in ``\nrf\samples\nrf9160\at_client\build_nrf9160_pca20035\zephyr`` if you build the secure version and in ``\nrf\samples\nrf9160\at_client\build_nrf9160_pca20035ns\zephyr`` if you build the non-secure version.
```

Comment:
I'm not familiar with SES environment for building, but I think only the non-secure output should be valid for the AT client? BSD lib can't run from secure context

Reply:
Generalizing this as follows:
You can find the output of the build, which includes the merged HEX file containing both the application and the SPM, in the ``zephyr`` sub-folder in the build directory.